chef/cookbooks/metasploitable/recipes/apache_struts.rb

```diff
@@ -0,0 +1,37 @@
+#
+# Cookbook:: metasploitable
+# Recipe:: apache_struts
+#
+# Copyright:: 2017, The Authors, All Rights Reserved.
+
+include_recipe 'metasploitable::tomcat'
+include_recipe 'metasploitable::java'
+
+file 'C:\Program Files\Apache Software Foundation\tomcat\apache-tomcat-8.0.33\conf\tomcat-users.xml' do
+  action :delete
+end
```

Comment:
This is not idempotent, the `cookbook_file` resource will override this file you don't need to delete it.